### ISSUE TITLE
Add. 14,726, foll. 1–86 | https://bl.vuexistapps.us/manuscript/620 (Part1)

### ISSUE THREAD

**Issue by Erfan-Syriaca** [srophe/britishLibrary-data#695]
### Categories of Manuscript Overview Problems

- [ ] Whitespace
- [ ] Typo(s)
- [ ] Incorrect content
- [ ] Missing content

### Problems with Physical Description

- Content pending
- Accompanying Material is empty. 

### Categories of Physical Description Problems

- [ ] Whitespace
- [ ] Typo(s)
- [ ] Incorrect content
- [X] Missing content

**Comment by mikey-tang16** [comment]
This was missed previously: 
A different hand inserted the words <foreign xml:lang="syr">ܝ̇ܠܕܬ ܐܠܗܐ</foreign> in the first chapter of the treatise by Elias the Harrānite, of Salamya. So I am adding this to a second hand (minor unspecified). 

As for the font problem in item 16, I wrapped the syriac with foreign labels. So I have
note, foreign xml:lang="syr" ܝ̇ܠܕܬ ܐܠܗ /foreign, foreign xml:lang="en" is added between the lines by a different hand. foreign note
Does this solve the display issue?
@wlpotter

**Comment by wlpotter** [comment]
Yes it should. Can you add the `@type="footnote"` to that note element, as well?

**Comment by mikey-tang16** [comment]
note element added
